Rdev/xgx_ordinal_reg.R

```diff
@@ -0,0 +1,78 @@
+#' @importFrom brms brm
+#' @importFrom brms 
+#' @export
+xgx_ordinal_regression_plot <- function(data = NULL,
```

Comment:
overall, the function name and file names should match.  And there should be one R file per function.

Reply:
Let's not focus on the files under Rdev for now